src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/BerConverter.cs

```diff
@@ -486,7 +486,9 @@ private static int EncodingByteArrayHelper(SafeBerHandle berElement, byte[] temp
 
         private static byte[] DecodingByteArrayHelper(SafeBerHandle berElement, char fmt, ref int error)
         {
+#pragma warning disable IDE0059 // TODO: https://github.com/dotnet/roslyn/issues/42761
             error = 0;
+#pragma warning restore IDEOO59
```

Comment:
This one can also be removed.  It's private and no call sites care about the value of error in the case of an exception nor try to observe error from multiple threads.

Reply:
Removed